docs/content/media/azure.md

```diff
@@ -0,0 +1,21 @@
+## Azure
+
+You can use [Microsoft Azure Blob Storage](https://azure.microsoft.com/services/storage/blobs/) to handle your image uploads in HedgeDoc.
+
+All you need to do is create a container with public access set to 'blob' and a [connection string](https://docs.microsoft.com/azure/storage/common/storage-account-keys-manage).
+
+It's possible to create the container with the [azure cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli), if you have a connection string, with the following command:
+
+```
+az storage container create --name NAME --public-access blob --connection-string "CONNECTION_STRING"
```

Comment:
Maybe the templates should be annotated in another way. Maybe like this:
```suggestion
az storage container create --name NAME --public-access blob --connection-string "<CONNECTION_STRING>"
```
This is especially relevant for the environment variables.

Reply:
Should this be changed for all placeholder or only that one?

Reply:
For all, maybe there is a better way but I think it's important to see clearly what is a placeholder.

Reply:
fixed